docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -119,6 +119,74 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Receiving events
+
+The changes in properties of the [`VideoPlayer`](#videoplayer) do not update the React state, therefore in order to display the information about the current state of the VideoPlayer it is necessary to listen to the [events](#videoplayerevents) it emits. There are a few ways to listen to events:
+
+- `useEvent` hook - this hook creates a listener that will return a stateful value that can be used in a component. It also cleans up automatically when the component unmounts.
+
+```tsx
+import { useEvent } from 'expo';
+import { useVideoPlayer } from 'expo-video';
+
+export default function ListenToStatus() {
+  const player = useVideoPlayer(videoSource);
+  const { status, error } = useEvent(player, 'statusChange', { status: player.status });
+
+  // Rest of the component...
+}
+```
+
+- `useEventListener` hook - built around the `Player.addEventListener` and `Player.removeEventListener` methods, creates an event listener with automatic cleanup.
+
+```tsx
+import { useEventListener } from 'expo';
+import { PlayerError, useVideoPlayer } from 'expo-video';
+import { useState } from 'react';
+
+export default function ListenToStatus() {
+  const player = useVideoPlayer(videoSource);
+  const [playerStatus, setPlayerStatus] = useState(player.status);
+  const [playerError, setPlayerError] = useState<PlayerError | null>(null);
+
+  useEventListener(player, 'statusChange', ({ status, error }) => {
+    setPlayerStatus(status);
+    setPlayerError(error);
+    console.log('Player status changed: ', status);
+  });
+
+  // Rest of the component...
+}
+```
+
+- `Player.addEventListener` method - this is the most flexible way to listen to events, but requires manual cleanup and more boilerplate code.
+
+```tsx
+import { useEffect, useState } from 'react';
+import { VideoPlayerEvents, useVideoPlayer, PlayerError } from 'expo-video';
+
+function ListenToStatus() {
+  const player = useVideoPlayer(videoSource);
+  const [playerStatus, setPlayerStatus] = useState(player.status);
+  const [playerError, setPlayerError] = useState<PlayerError | null>(null);
+
+  useEffect(() => {
+    const listener: VideoPlayerEvents['statusChange'] = ({ status, error }) => {
+      setPlayerStatus(status);
+      setPlayerError(error);
+      console.log('Player status changed: ', status);
+    };
+    player.addListener('statusChange', listener);
```

Comment:
It might be better to just inline the listener, then you don't need to import `VideoPlayerEvents` and the IntelliSense works nicely as you type – once you finish typing the event name, you'll get all the info about listener parameters.

```suggestion
    player.addListener('statusChange', ({ status, error }) => {
      setPlayerStatus(status);
      setPlayerError(error);
      console.log('Player status changed: ', status);
    });
```